apps/fluent-tester/src/TestComponents/Shadow/ShadowTest.tsx

```diff
@@ -13,12 +13,10 @@ const shadowSections: TestSection[] = [
   },
   {
     name: 'Shadows on Button Examples',
-    testID: SHADOW_TESTPAGE,
```

Comment:
Was curious why we're removing this?

Reply:
There should only be one instance of this testID per page, which is usually assigned to the first test section. Having multiple testIDs assigned to multiple components is a no-no I believe.

Reply:
This testID is just a way to determine if we're on the right page or not. It's only necessary to have it one the first one (top of the page), but it wouldn't cause any ill-effect if it's on multiple. Just as long as it's on the first one.